src/main/app/src/app/admin/parameter-edit/parameter-edit.component.html

```diff
@@ -87,16 +87,6 @@
                                         {{humanTaskParameter.planItemDefinition.naam}}
                                     </mat-panel-title>
                                 </mat-expansion-panel-header>
-                                <div fxLayout="row wrap">
-                                    <mat-form-field fxFlex="33" fxFlex.lt-md="100" appearance="standard" floatLabel="always">
-                                        <mat-label>{{'formulierDefinitie' | translate}}</mat-label>
-                                        <mat-select placeholder="{{'formulierDefinitie.-kies-' | translate}}"
-                                                    [formControl]="getHumanTaskControl(humanTaskParameter, 'formulierDefinitie')" [compareWith]="compareObject">
-                                            <mat-option *ngFor="let formDef of formulierDefinities | async"
-                                                        [value]="formDef">{{ 'formulierDefinitie.' + formDef | translate }}</mat-option>
-                                        </mat-select>
-                                    </mat-form-field>
-                                </div>
                                 <div fxLayout="row wrap">
```

Comment:
op het ParameterEditComponent zit nog een variabele: formulierDefinities. Deze is niet meer nodig toch? En ook niet de code om dat op te halen en niet in de service enz.

Reply:
Verwijderd